explorer/src/providers/accounts/rewards.tsx

```diff
@@ -111,13 +115,20 @@ async function fetchRewards(
   const getInflationReward = async (epoch: number) => {
     try {
       const result = await connection.getInflationReward([pubkey], epoch);
-      return result[0];
+      if (result[0] !== null) {
+        return result[0];
+      }
     } catch (error) {
-      if (cluster !== Cluster.Custom) {
+      if (
+        cluster !== Cluster.Custom &&
+        !error.message.match(/Block not available/)
+      ) {
         reportError(error, { url });
       }
     }
-    return null;
+    return {
+      epoch,
+    };
```

Comment:
Shouldn't the return value be different between a null response and a failed request?